coral/src/app/components/Modal.tsx

```diff
@@ -15,7 +15,7 @@ type ModalProps = {
   title: string;
   subtitle?: string;
   close?: () => void;
-  primaryAction: ModalAction;
+  primaryAction?: ModalAction;
```

Comment:
Why make the `primaryAction` optional? It's not needed for this PR, is it? If we leave it it, there should be a test added for it.

Reply:
Oops yeah this was a thing I used while developing, but is not needed currently.

Reply:
💡

Reply:
[37915ea](https://github.com/Aiven-Open/klaw/pull/2315/commits/37915eac2db839762ddced232b244393659f23c6)